specification/nginx/resource-manager/NGINX.NGINXPLUS/preview/2021-05-01-preview/swagger.json

```diff
@@ -1179,6 +1201,9 @@
       "properties": {
         "enableDiagnosticsSupport": {
           "type": "boolean"
+        },
+        "logging": {
+          "$ref": "#/definitions/NginxLogging"
```

Comment:
Is this change on 2021-05-01-preview intended? if so, please request breaking changes approval

Reply:
I reverted the change for the preview version. So it won't have breaking change. Thanks.